### PR TITLE
DN5865 - list sandboxes

### PR DESCRIPTION
**Pull request by mjbradford89**
https://polyswarm.atlassian.net/browse/DN-5865

### CODE REVIEW

File: src/polyswarm_api/resources.py [hunk 1]
```diff
@@ -984,10 +984,19 @@ def sandbox(cls, api, sha256):
...
-                'url':
-                '{}/consumer/submission/{}/sandbox/{}'.format(api.uri, api.community, sha256)
+                'method': 'POST',
+                'url': '{}/consumer/submission/{}/sandbox/{}'.format(api.uri, api.community, sha256)
+            },
+            result_parser=cls,
+        ).execute()
+
+    @classmethod
+    def sandbox_list(cls, api):
```
Comment (sbneto) [inline src/polyswarm_api/resources.py:994]: Please use the default routes there, check the live hunts on how to do that, we have listing methods there. We should try to use the default stuff as much as possible.

Comment (sbneto) [reply src/polyswarm_api/resources.py:994]: We can jump in a call if you think that will help.

Comment (mjbradford89) [reply src/polyswarm_api/resources.py]: Sure, makes sense. I made the change but had to override `_endpoint` to format the uri with the community: https://github.com/polyswarm/polyswarm-api/pull/190/files#diff-5b575475ce5b098eb1a647726381758cb9b5f86dc914ce3eab3a7db8d25dc3b3R995

Not sure if there is a better way to do that but this seemed the most simple.

**Review by alanjds — DISMISSED** [review]

File: src/polyswarm_api/core.py [hunk 2]
```diff
@@ -328,6 +329,8 @@ def _delete_endpoint(cls, api, **kwargs):
     @classmethod
     def _params(cls, method, *param_keys, **kwargs):
         params = {}
+        if cls.QUERY_PARAM_SCOPE:
+            params = {scope: kwargs.get(scope) for scope in cls.QUERY_PARAM_SCOPE}
```
Comment (sbneto) [inline src/polyswarm_api/core.py:333]: I am not understanding where this `param` variable get's used, am I missing something?

**Review by mjbradford89 — COMMENTED** [review]
lgtm

**Pull request merged**